kong/templates/kong_defaults.lua

```diff
@@ -77,6 +77,7 @@ declarative_config = NONE
 db_update_frequency = 5
 db_update_propagation = 0
 db_cache_ttl = 0
+db_cache_neg_ttl = 0
```

Comment:
```suggestion
db_cache_neg_ttl = NONE
```

so that the code inherits `db_cache_ttl` when not given